readme.md

```diff
@@ -64,39 +71,40 @@ $ npm -v
 
 * **Your Answer:**
 
----
+A "dot-file" is a hidden file.  A ".gitignore" file is a list of files to ignore when pushing files to your remote repo.
+
 
 - [ ] Create an `index.js` file with the following contents: `console.log('Hello, Node!')`
 
 * **Question:** From the command line, how can you run this file?
 
 * **Your Answer:**
 
----
+node index.js
 
 - [ ] Run `npm test` from the command line
 
 * **Question:** What happens and how is this related to what is in the `package.json` file? 
 
 * **Your Answer:**
 
----
+Running npm test will run the script "test" in the package.json file, added when I ran an npm init -y
 
 - [ ] Create a new "script" command called "start" that has the following value: `node index.js`
 
 * **Question:** What will you enter on the command line to run that script?
 
 * **Your Answer:**
 
----
+npm start
 
 - [ ] Change the name of your "start" script to "my-file"
 
 * **Question:** The same pattern will not work to try and run this script. How can you successfully get this script to run?
 
 * **Your Answer:**
 
----
+Change it back to start
```

Comment:
You can define your own scripts. To run this one, you would type `npm run my-file`.